devfile.yaml

```diff
@@ -22,48 +25,7 @@ components:
   - name: outerloop-deploy
     kubernetes:
       uri: outerloop-deploy.yaml
-  - name: runtime
-    container:
-      image: registry.access.redhat.com/ubi8/nodejs-14:latest
```

Comment:
It is now pull from the devfile registry. The nodejs stack: https://registry.devfile.io/devfiles/nodejs, contains all the content I deletes from the sample's devfile. When tools parse this devfile, it will merge the parent devfile into this devfile.

Reply:
Yes. you just need to update the stack, the sample will pick it up.